### PR TITLE
i18n(zh-cn): Update solid-js.mdx

### PR DESCRIPTION
**Pull request by huyikai**
#### Description (required)

Update solid-js.mdx

#### Related issues & labels (optional)

- #5978 
- #6005 
- #6109

### CODE REVIEW

**Review by liruifengv — APPROVED** [review]
LGTM

**Comment by sarah11918** [comment]
@huyikai @liruifengv - This file had additions today (new section at the bottom). Would you like to add those changes here, too?

**Comment by huyikai** [comment]
> @huyikai @liruifengv - This file had additions today (new section at the bottom). Would you like to add those changes here, too?

Yes, We'll let you know when it's updated.

**Comment by huyikai** [comment]
@liruifengv Updated, please review

File: src/content/docs/zh-cn/guides/integrations-guide/solid-js.mdx [hunk 1]
```diff
@@ -6,59 +6,87 @@ githubIntegrationURL: 'https://github.com/withastro/astro/tree/main/packages/int
 category: renderer
 i18nReady: true
 ---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 此 **[Astro 集成][astro-integration]**为你的 [SolidJS](https://www.solidjs.com/) 组件启用服务器端渲染和客户端水合。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/integrations-guide/solid-js.mdx:11]:
```suggestion
此 **[Astro 集成][astro-integration]** 为你的 [SolidJS](https://www.solidjs.com/) 组件启用服务器端渲染和客户端水合。
```

**Review by liruifengv — APPROVED** [review]
Perfect!

**Comment by huyikai** [comment]
@sarah11918 ready

**Pull request merged**